UI/script.js

```diff
@@ -61,6 +61,17 @@ function showSlides() {
     setTimeout(showSlides, 2000); // Change image every 2 seconds
 }
 
+//topnav
+function myFunction1() {
+    var x = document.getElementById("myTopnav");
+    if (x.className === "topnav") {
+        x.className += " responsive";
+    } else {
+        x.className = "topnav";
```

Comment:
You can also add and remove css classes using `.classList.add` and `.classList.remove` method.

See this:
https://www.w3schools.com/howto/howto_js_add_class.asp

Reply:
Thanks. I will look into it